doc/introduction.rst

```diff
@@ -29,6 +29,11 @@ Examples of geometry design variables might include:
 The set of all geometry design variables and their effects on the baseline geometry is called a *parameterization*.
 Parameterizations are not unique, and choosing a good one is both an art and a science.
 
+There are two primary approaches to geometry parameterization: generative and deformative.
+In pyGeo, we primarily use a free form deformation (FFD) approach, which is deformative.
+A deformative parameterization does not require as detailed of a parameterization as a generative approach, such as CAD, does.
```

Comment:
Mmm.. I think I am misunderstanding what these blue points represent. Are they the "embedded volume" that's returned after #133? to me they look like an extra "random" pointset and the shaded area is already effective at showing the extent of the FFD volume. We should clarify where they is coming from because we previously refer to the blue triangles that define the cylinder as pointset, and they are the "deformation target".
Hope I am not making things more confusing, we can talk offline